x/blob/types/blob_tx.go

```diff
@@ -33,7 +35,11 @@ func NewBlob(ns appns.Namespace, data []byte, shareVersion uint8) (*blob.Blob, e
 
 // ValidateBlobTx performs stateless checks on the BlobTx to ensure that the
 // blobs attached to the transaction are valid.
-func ValidateBlobTx(txcfg client.TxEncodingConfig, bTx blob.BlobTx) error {
+func ValidateBlobTx(txcfg client.TxEncodingConfig, bTx *blob.BlobTx, subtreeRootThreshold int) error {
+	if bTx == nil {
+		return errors.New("nil blob tx")
```

Comment:
[Suggestion] All the errors returned from this function are named, so it would be beneficial to consider naming this particular error as well, to maintain consistency.